### ISSUE TITLE
Table editing in a standalone note window doesn't work on Windows

### ISSUE THREAD

**Issue by mrtcode** [zotero/zotero#2564]
https://forums.zotero.org/discussion/96604/unable-to-enter-text-in-table-in-separate-window#latest

**Comment by mrtcode** [comment]
Seems like a Firefox bug. It's quite similar to the resize handle issue, because in both cases there is a strange displacement. Difficult to debug further because dev tools doesn't work well with external windows (node inspector is blank for me).

**Comment by dstillman** [comment]
> dev tools doesn't work well with external windows (node inspector is blank for me)

Yeah, same for me, though innerHTML on specific nodes works from the console.
